### ISSUE TITLE
Electronic cigarette or vape use

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10215]
**Preferred term label:**
Electronic cigarette or vape use

**Synonyms**
Vaping

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C4505216
https://uts.nlm.nih.gov/uts/umls/concept/C4083280

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lipoid pneumonia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as Electronic cigarette exposure history
HP:0430032
